### PR TITLE
Complete lint for cmd/influx

### PR DESCRIPTION
**Pull request by pablolmiranda**
- [ ] CHANGELOG.md updated
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](http://influxdb.com/community/cla.html) (if not already signed)

Part of #4098.

### CODE REVIEW

**Comment by otoolep** [comment]
Thanks @pablolmiranda -- you must sign the CLA before we can merge this. https://influxdb.com/community/cla.html

File: cmd/influxd/run/server.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/influxdb/influxdb/services/udp"
 	"github.com/influxdb/influxdb/tcp"
 	"github.com/influxdb/influxdb/tsdb"
+	// What is the reason to have a blank import here?
```
Comment (otoolep) [inline cmd/influxd/run/server.go]: This is to initialize the engine packages.

Comment (otoolep) [inline cmd/influxd/run/server.go]: So if lint requires this to be commented, please add the comment above.

**Comment by pablolmiranda** [comment]
@otoolep CLA signed.

**Comment by otoolep** [comment]
+1.

@corylanou

**Comment by corylanou** [comment]
Looks good.  Needs a rebase to resolve the conflicts and then we can merge. Thanks!

**Comment by pablolmiranda** [comment]
@otoolep @corylanou can you double check this PR?

**Comment by otoolep** [comment]
Looks good @pablolmiranda -- can you please squash your commits and I will then merge. A comment-only change like this should be a single commit.

Also please send me an e-mail with your address so we can send you a shirt.

**Comment by otoolep** [comment]
Before you squash you should actually rebase your branch against our master.

`git pull --rebase influxdb master`

File: cmd/influx/cli/cli.go [hunk 2]
```diff
@@ -255,14 +262,17 @@ func (c *CommandLine) Connect(cmd string) error {
 		return fmt.Errorf("Could not create client %s", err)
 	}
 	c.Client = cl
-	if _, v, e := c.Client.Ping(); e != nil {
+	if _, _, e := c.Client.Ping(); e != nil {
 		return fmt.Errorf("Failed to connect to %s\n", c.Client.Addr())
-	} else {
-		c.ServerVersion = v
```
Comment (corylanou) [inline cmd/influx/cli/cli.go]: why was this removed?

**Comment by corylanou** [comment]
One question otherwise looks good.

File: cmd/influx/cli/cli.go (hunk 2 above)
Comment (otoolep) [inline cmd/influx/cli/cli.go]: Good catch @corylanou

Comment (pablolmiranda) [inline cmd/influx/cli/cli.go]: __golint__ ask to move it out of the else block. But I removed by accident. Fixed now.
